plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -573,33 +581,6 @@ adjustTxOut params txOut = do
     else pure ([], txOut)
 
 
--- | Add the remaining balance of the total value that the tx must spend.
---   See note [Balance of value spent]
-addMissingValueSpent
-    :: ( MonadReader (ScriptLookups a) m
-       , MonadState ConstraintProcessingState m
-       , MonadError MkTxError m
-       )
-    => m ()
-addMissingValueSpent = do
-    missing <- gets totalMissingValue
-    if Value.isZero missing
-        then pure ()
-        else do
-            -- add 'missing' to the transaction's outputs. This ensures that the
-            -- wallet will add a corresponding input when balancing the
-            -- transaction.
-            -- Step 4 of the process described in [Balance of value spent]
-            pkh <- asks slOwnPaymentPubKeyHash >>= maybe (throwError OwnPubKeyMissing) pure
```

Comment:
I think this means we can remove `OwnPubKeyMissing`?

Reply:
I think so, I'm doing it.